main.go

```diff
@@ -54,13 +55,17 @@ func HandleConfigFile(filePath string) (config.ConfigReader, error) {
 
 	c := viper.New()
 
+	// map settings such as foo.bar to FOO_BAR environment keys
+	c.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+
 	//Allow AWS URI endpoint to be set by environment variable
 	c.BindEnv(SettingAwsURI, "AWS_URI")
 
 	//Allow aws keyid, aws token, aws secret to be read by viper
 	c.BindEnv(SettingAwsAuthKeyId, "AWS_ACCESS_KEY_ID")
 	c.BindEnv(SettingAwsAuthSecret, "AWS_SECRET_ACCESS_KEY")
 	c.BindEnv(SettingAwsAuthToken, "AWS_SESSION_TOKEN")
+	c.BindEnv(SettingAwsS3Region, "AWS_S3_REGION")
```

Comment:
I'm not sure I fully understood both tickets, but in [MEN-747](https://tracker.mender.io/browse/MEN-747?focusedCommentId=77578&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-77578) you mentioned:
> ah, sorry if this was not that clear, was assuming we will still maintain functionality of overriding config settings with environment variables as we are able to do today. As you said this is very convenient, especially in out current setup. 

Removing these options will not preserve config through environment functionality.